msal/src/Microsoft.Identity.Client/Account.cs

```diff
@@ -61,11 +60,20 @@ public Account(AccountId homeAccountId, string username, string environment)
 
         public override string ToString()
         {
-            return String.Format(
```

Comment:
It looks like the parameterless ctor is only used by tests. Could we get rid of it instead? This way HomeAccountId would be guaranteed not null.

Reply:
great idea. thanks.

Reply:
@bgavrilMS I updated all the unit tests.